### PR TITLE
Adding `geoindex-rs` (A Python package)

### PR DESCRIPTION
**Pull request by kylebarron**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kylebarron** [comment]
@conda-forge/help-python, ready for review!

**Comment by kylebarron** [comment]
@conda-forge/help-python, ready for review!

**Comment by kylebarron** [comment]
@conda-forge/help-python, ready for review!

**Comment by ocefpaf** [comment]
@conda-forge/help-rust

**Comment by kylebarron** [comment]
@ocefpaf this is a Python package, not a Rust package

File: recipes/geoindex_rs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: geoindex_rs
```
Comment (xhochy) [inline recipes/geoindex_rs/meta.yaml:4]: This should probably be a `-` hyphen
```suggestion
  name: geoindex-rs
```

**Comment by ocefpaf** [comment]
> @ocefpaf this is a Python package, not a Rust package

It has a rust extension, no?

**Comment by xhochy** [comment]
We could do a `conda-forge/help-python-rust` team like we have for C++ but for now it would simply be me again 🤷

**Comment by ocefpaf** [comment]
> We could do a `conda-forge/help-python-rust` team like we have for C++ but for now it would simply be me again 🤷

I'm watching your reviews of rust packages to learn and I'll help soon.

**Comment by xhochy** [comment]
> I'm watching your reviews of rust packages to learn and I'll help soon.

Most of it is checking for license bundling 🥲

**Comment by ocefpaf** [comment]
> I didn't realize that there intended to be a separate review for Rust-Python packages as for Python packages. (#27209 was also a Rust-Python library

That is why I waited a lot for someone else to review that one but we are few and our volunteer hours limited, so I got that for you.

**Comment by kylebarron** [comment]
I see; well, I appreciate your time

**Review by xhochy — APPROVED** [review]

**Pull request merged**